lib/util.js

```diff
@@ -53,7 +53,9 @@ exports.getLabels = function (labelNames, args) {
 	}
 
 	if (labelNames.length !== args.length) {
-		throw new Error('Invalid number of arguments');
+		throw new Error(
+			`Invalid number of arguments: "${args.join()}" for label names: "${labelNames.join()}".`,
```

Comment:
I wonder if something like this is even more helpful?

```suggestion
			`Invalid number of arguments (${args.length}): "${args.join(', ')}" for label names (${labelNames.length}): "${labelNames.join(', ')}".`,
```

?

Reply:
💪 I like this suggestion, I'll add it.